static/styles/compose.css

```diff
@@ -1,3 +1,9 @@
+/*
+Common background color
+TODO: import from a common place
+*/
+$background_grey: hsl(0, 0%, 94%);
```

Comment:
Agreed. This may be a good discussion for #frontend -- we should possibly initialize a near-empty file `app_colors.css` that's imported everywhere as a prep PR, so that things like this can just add lines to that.